clang/lib/CodeGen/CMakeLists.txt

```diff
@@ -148,6 +148,7 @@ add_clang_library(clangCodeGen
   clangAST
   clangAnalysis
   clangBasic
+  clangCIRFrontendAction
```

Comment:
You need to isolate this addition under cmake generated CIR macro, because we can build clang without CIR support, there are examples around.

Reply:
Done